_includes/layouts/blank.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en-US">
-  {% include head.html title=page.title %}
-  <body{% if layout.body_class %} class="{{ layout.body_class }}"{% endif %}>
+  {% include head.html, title: page.title %}
```

Comment:
@emersonthis Whoops, just found a bug I missed before. Compare page titles on permalinks...

- https://www.pwastats.com/2020/04/rooted-objects/
- https://deploy-preview-206--pwastats.netlify.app/2020/04/rooted-objects/

Reply:
I have this fixed on a local branch. But the same bug for the tag archive pages is proving harder to solve...

Reply:
solved